frontend/src/components/Metrics/CustomMetrics.tsx

```diff
@@ -353,7 +353,7 @@ class CustomMetricsComponent extends React.Component<Props, MetricsState> {
 
 const mapStateToProps = (state: KialiAppState) => {
   return {
-    jaegerIntegration: state.jaegerState.info ? state.jaegerState.info.integration : false,
+    jaegerIntegration: state.tracingState.info ? state.tracingState.info.integration : false,
```

Comment:
If the state property is generic (tracingState), the variable should be renamed to tracingIntegration